test/integration/full/landmark-no-more-than-one-main/frames/level1.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html id="pass2">
+<head>
+	<meta charset="utf8">
+	<script src="/axe.js"></script>
+</head>
+<body>
+	<p>No main content should still pass</p>
```

Comment:
This rule is making sure that there is no more than one main landmark in any one document (e.g. the container document or any of its iframes).   In general the ARIA authoring practices recommends only one main landmark per page.  But there is an exception for web sites that use iframes, for example web applications like portals often have content coming from other servers and services and are included in a web page within an iframe.  In this case where there is a high probability of distributed authorship, the ARIA Authoring Practices allows the main document and each iframe to have a main landmark.   

Tis rule is just checking that the main document or any documents in iframes have no more than one main landmark.

Hope this makes sense,
Jon